lib/src/syntax/api.ts

```diff
@@ -15,4 +15,5 @@ export function api(config: ApiConfig) {
 export interface ApiConfig {
   /** Name of the API. This should be the name of the service that is being documented */
   name: string;
+  version?: string;
```

Comment:
```suggestion
  /** Version of this document */
  version?: string;
```

Reply:
Ugh.... Sorry about that, I always forget docs.